core/src/main/scala/org/apache/spark/deploy/rest/RestSubmissionServer.scala

```diff
@@ -93,6 +94,14 @@ private[spark] abstract class RestSubmissionServer(
     contextToServlet.foreach { case (prefix, servlet) =>
       mainHandler.addServlet(new ServletHolder(servlet), prefix)
     }
+    if(masterConf.getBoolean("spark.rest.csrf.enable", false)) {
```

Comment:
I'm not familiar with this but the question is should the config be documented?  I assume this is something you want the end user to have the option of using?  

if this is only being used by the private Rest apis should it be true by default or what exactly are the ramifications of that?